src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+import { PeopleContext } from '../../peopleContext';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    setError(false);
+    setLoading(true);
+
+    getPeople()
+      .then(res => setPeople(res))
```

Comment:
It's better to handle data transformation in a service layer (in this case inside the 'getPeople' function). This way, the component doesn't need to know about the underlying data format and we keep the separation of concerns principle.